src/blockchain/contracts/erc20.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from blockchain.contracts.base_interface import ContractInterface
+from eth_typing import ChecksumAddress
+
+logger = logging.getLogger(__name__)
+
+
+class ERC20Contract(ContractInterface):
+    abi_path = './interfaces/ERC20.json'
+
+    def balance_of(self, address: ChecksumAddress) -> int:
```

Comment:
In depositor we don't use block_identifier kwarg, but let's add it everywhere to keep consistency.

Reply:
Done